src/cuga/backend/tools_env/registry/registry/api_registry_server.py

```
@@ -140,7 +140,19 @@ async def _get_or_create_registry(
     manager = MCPManager(config=services)
-    reg = ApiRegistry(client=manager)
+    
+    # Get policy storage if policy system is enabled
+    policy_storage = None
+    if settings.policy.enabled:
+        try:
+            from cuga.backend.cuga_graph.policy.storage import PolicyStorage
+            policy_storage = PolicyStorage()
+            await policy_storage.connect()
+            logger.info("✅ Connected policy storage for ToolGuardRuntime")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)